charts/seed-bootstrap/charts/loki/values.yaml

```
@@ -51,7 +51,8 @@ replicas: 1
 resources:
   loki:
     limits:
-      memory: 10Gi
+#TODO (vlvasilev) after eliminating the Loki OOM Kill issue return it to 10Gi
```

By this issue do you mean a bug on our side (memory leak in the curator for example)? If yes, can you please link the issue.

```suggestion
      # TODO(vlvasilev): After fixing the following issue <issue-link>, set the memory limit to 10Gi.
      memory: 3Gi
```

---

Still, there is no open issue. I am working on that problem now.

---

So is it a different issue from https://github.com/gardener/logging/issues/119? If yes, please open a new issue and link it in the TODO so we know exactly after which issue fix we are safe to increase the limit again.